CONTRIBUTING.md

```diff
@@ -684,41 +684,45 @@ icon.png
   ```
 
 ## Quickstart Preview
+⚠️  There is currently no way to preview Quickstarts locally as Jumpstart no longer owns the public I/O website (newrelic.io).
```

Comment:
I think the issue with the preview not working stems from our new tech stack (Drupal) vs the Gatsby-based approach we had before. Furthermore, because this is a public-facing repository, including our team name might not be helpful to the general public (what's a JumpStart?)

Reply:
Yeah, it apparently used to work? Unless you mean that we recently switched to Drupal (I have no idea).

Good point on the team name!